README.md

```diff
@@ -1,10 +1,9 @@
 # Installation
 
 ```julia
-Pkg.clone("https://github.com/AStupidBear/SNN.jl.git")
-Pkg.build("SNN")
```

Comment:
I removed the build code since it didn't seem necessary, and also didn't seem like a good idea to me personally (if we need `Utils.jl`, we should depend on it explicitly).